packages/mdx/create-element.js

```diff
@@ -0,0 +1,59 @@
+const React = require('react')
+const {withMDXComponents} = require('@mdx-js/tag')
+
+const TYPE_PROP_NAME = '__MDX_TYPE_PLEASE_DO_NOT_USE__'
```

Comment:
In case someone sees this and is concerned that this is some kind of a bug, we could make it clearer that this is for internal use only, something like `__INTERNAL_MDX_TYPE__ ` or `__INTERNAL_MDX_TYPE_DO_NOT_USE_DIRECTLY__`.